src/Sylius/Bundle/ShopBundle/index.js

```
@@ -11,21 +11,36 @@ const path = require('path');
+        Encore.reset();
+
+        return shopConfig;
+    }
+
+    /**
+     * For a ready-to-use Stimulus bridge. Should be used only for sylius/sylius tests
+     * For instances started with Sylius-Standard < 2.0.4, it'll still be used unless upgrading webpack.config.js
+     * to use the method above getBaseWebpackConfig()
+     */
     static getWebpackConfig(rootDir) {
+        this._prepareWebpackConfig(rootDir);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)